### PR TITLE
api: removing dependency on hardcoded value when onboarding consumer

### PR DESCRIPTION
**Pull request by rchikatw**
when onboarding a new consumer using API parameters are changed the same change needs to be implement here. Now no longer need to pass the hard coded value while onboarding a consumer.

### CODE REVIEW

**Comment by rchikatw** [comment]
@Madhu-1 Can you review this

File: go.mod [hunk 1]
```diff
@@ -2,48 +2,46 @@ module github.com/red-hat-storage/ocs-client-operator
 
 go 1.18
 
-replace k8s.io/client-go => k8s.io/client-go v0.25.2
+replace (
+	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3 // required by rook v1.7
```
Comment (Madhu-1) [inline go.mod:6]: we are not using Rook 1.7 even in ocs-operator why do we need this one?

File: go.mod [hunk 2]
```diff
@@ -2,48 +2,46 @@ module github.com/red-hat-storage/ocs-client-operator
...
+	k8s.io/client-go => k8s.io/client-go v0.26.4
+	vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
```
Comment (Madhu-1) [inline go.mod:8]: why do we need this one? i don't see this is required for ocs-operator as well.

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

**Comment by Madhu-1** [comment]
@rchikatw please check commitlint Ci failure https://github.com/red-hat-storage/ocs-client-operator/actions/runs/5622866814/job/15236315686?pr=32

File: go.mod (hunk 2 above)
Comment (rchikatw) [reply go.mod:8]: this is required for nooba-operator 5, If we do not use we will get this error while building.

`... go: downloading github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
go: github.com/openshift/cloud-credential-operator@v0.0.0-20210716145643-0ee1808fb8ed requires
    github.com/openshift/library-go@v0.0.0-20200911100307-610c6e9e90b8 requires
    vbom.ml/util@v0.0.0-20180919145318-efcd4e0f9787: unrecognized import path "vbom.ml/util": https fetch: Get "https://vbom.ml/util?go-get=1": dial tcp: lookup vbom.ml: no such host
ritesh:ocs-client-operator$`

File: go.mod (hunk 1 above)
Comment (rchikatw) [reply go.mod:6]: @iamniting suggested this to add, if not added build is failing.

Comment (Madhu-1) [reply go.mod:6]: update the comment to point to rook 1.12 recent release?

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**